rust/repo/{{ cookiecutter.repo_name }}/.github/CODEOWNERS

```diff
@@ -1,11 +1,17 @@
-/Cargo.toml
-/Cargo.lock
-/LICENSE
-/deny.toml
+# These lines impact repository security
+/.github/CODEOWNERS        @nick-mobilecoin @jcape
+/.github/settings.yml      @nick-mobilecoin @jcape
```

Comment:
:question: Do we want me as an owner to *all* repos?
I'm wondering if longer term this should be one of the prompts 
```
repo_owners[@nick-mobilecoin @jcape]:
```

Reply:
We could always ensure `@jcape` is on there as an escape hatch in case someone puts in invalid names. IDK maybe it's safer to leave it hard coded.

Reply:
Yeah, that seems reasonable, fixed.